### DISCUSSION TITLE
[suggestion] should api `/detect` support `pixel_perfect`?

### DISCUSSION THREAD

**Discussion by zombieyang** [Mikubill/sd-webui-controlnet#1741] [Ideas]
AFAIK, `pixel_perfect` is for preprocessing image. But when I want to invoke `/detect` I found that the new option `pixel_perfect` is not supported. Should it be added to `/detect` api?

**Comment by huchenlei** [comment]
See https://github.com/Mikubill/sd-webui-controlnet/issues/1144
